src/test/java/net/openhft/chronicle/queue/DirectoryUtils.java

```diff
@@ -31,6 +31,7 @@ public static File tempDir(String name) {
         String replacedName = name.replaceAll("[\\[\\]\\s]+", "_").replace(':', '_');
         final File tmpDir = new File(OS.getTarget(), replacedName + "-" + uniqueId());
         DeleteStatic.INSTANCE.add(tmpDir);
+        tmpDir.deleteOnExit();
```

Comment:
It looks as though the previous line _should_ take care of this already?

Reply:
just best practice to set these things as deleteOnExit as shutdown hooks may not get run